docs/quick-starts/bandit-quickstart.md

```diff
@@ -0,0 +1,188 @@
+
+Give the Bandit a descriptive human readable name. Select the entity and input the name of table in your warehouse where you are logging bandit actions from Step 2. Choose a metric to optimize the Bandit for and how long you want the lookback window to be. The lookback window defines the time period the Bandit looks for changes in the optimization metric for subjects who were assigned.
+
+![Setup Bandit step](/img/bandit/bandit-qs-1.png)
+
+### 4. Add variations
+To evaluate the impact of the Contextual Bandit, we run an experiment with exactly two variations: the bandit and a control. This control variation should be the "status quo" that you implement on your end. A simple example is a fixed action.
+
+![Add variations step](/img/bandit/bandit-qs-2.png)
+
+### 5. Initialize the SDK
+
+Choose the Eppo SDK that fits in your stack. You'll need to initialize the SDK in your app and create an Eppo client. Here is an example in Java:
+
+```java
+EppoClientConfig config = EppoClientConfig.builder()
+        .apiKey("<api-key>")
+        .assignmentLogger((data) -> System.out.println(data.toString()))
+        .build();
+EppoClient eppoClient = EppoClient.init(config);
+```
+Note, here is where you use the SDK key generated in step 1.
+
+### 6. Embed the Bandit in your code
+
+Once the SDK is initialized, use getStringAssignment() to check what variation a user should see. When requesting an assignment from a flag with a bandit, the set of actions and their attributes are provided as an additional argument.
+
+For example, in the Java SDK, the call may look like:
+```java
+// Flag that has a bandit variation
+String banditTestFlagKey = "bandit-test";
+
+// Subject information--same as for retrieving simple flag or experiment assignments
+String subjectKey = username;
+EppoAttributes subjectAttributes = userAttributes;
+
+// Action set for bandits
+Map<String, EppoAttributes> actionsWithAttributes = Map.of(
+  "dog", new EppoAttributes(Map.of(
+    "legs": EppoValue.of(4),
+    "size": "large"
+  )),
+  "cat", new EppoAttributes(Map.of(
+    "legs": EppoValue.of(4),
+    "size": "medium"
+  )),
+  "bird", new EppoAttributes(Map.of(
+    "legs": EppoValue.of(2),
+    "size": "medium"
+  )),
+  "goldfish", new EppoAttributes(Map.of(
+    "legs": EppoValue.of(0),
+    "size": "small" 
+  ))   
+);
+
+Optional<String> banditAssignment = eppoClient.getStringAssignment(subjectKey, flagKey, subjectAttributes, actionsWithAttributes);
+```
+
+If the actions don't have attributes for the context, you can simply provide a set of actions without attributes:
+```java
+Set<String> actions = Set.of("dog", "cat", "bird", "goldfish");
+
+Optional<String> banditAssignment = eppoClient.getStringAssignment(subjectKey, flagKey, subjectAttributes, actions);
+```
+
+Note that the `getAssignment` methods in Eppo are deterministic, meaning that they will always return the same variant for a given subject (e.g., user) for the life of the bandit.
```

Comment:
That's probably surprising to many people familiar with Bandit. I’m not sure if this would need a clarification like, “When Bandit decides which options are preferable, new it will expose  subjects to those options.” 
You might also need to explain: “Once Bandit has converged on the best option, you will be able to A/B test and roll out that option to all subjects.” for our clients who have mostly permanent users and don’t want most of them to get stuck with sub-optional options.